bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -74,6 +74,53 @@ spec:
+                          description: nodeSelector will allow/deny traffic to the
+                            Kubernetes node IP of selected nodes. If this is, set
+                            cidrSelector and DNSName must be unset.
+                          properties:
+                            matchExpressions:
+                              description: matchExpressions is a list of label selector
+                                requirements. The requirements are ANDed.
+                              items:
+                                description: A label selector requirement is a selector
+                                  that contains values, a key, and an operator that
+                                  relates the key and values.
+                                properties:
+                                  key:
+                                    description: key is the label key that the selector
+                                      applies to.
+                                    type: string
+                                  operator:
+                                    description: operator represents a key's relationship
+                                      to a set of values. Valid operators are In,
+                                      NotIn, Exists and DoesNotExist.
+                                    type: string
+                                  values:
+                                    description: values is an array of string values.
+                                      If the operator is In or NotIn, the values array
+                                      must be non-empty. If the operator is Exists
+                                      or DoesNotExist, the values array must be empty.
+                                      This array is replaced during a strategic merge
+                                      patch.
+                                    items:
+                                      type: string
+                                    type: array
+                                required:
+                                  - key
+                                  - operator
+                                type: object
+                              type: array
+                            matchLabels:
+                              additionalProperties:
+                                type: string
+                              description: matchLabels is a map of {key,value} pairs.
+                                A single {key,value} in the matchLabels map is equivalent
+                                to an element of matchExpressions, whose key field
+                                is "key", the operator is "In", and the values array
+                                contains only "value". The requirements are ANDed.
+                              type: object
+                          type: object
                       type: object
                       minProperties: 1
```

Comment:
in the upstream PR the min and max properties were removed since before it was only cidrSelector versus dnsName and now we here we haven't removed them although I feel this is the correct behaviour, not what went in upstream, left a comment on that PR asking for more clarification:  https://github.com/ovn-org/ovn-kubernetes/pull/3002#discussion_r1121251873